recipe/meta.yaml

```diff
@@ -33,7 +39,7 @@ requirements:
     - jupyter_client >=6.1.1
     - jupyter_core >=4.6.1
     - jupyter_server >=1.8
-    - nbconvert >=5
+    - nbconvert{{ "" if use_pandoc != "no" else "-core" }} >=5
```

Comment:
probably need some kind of test that excites this so the failure mode is easy to identify...

Reply:
I don't think there is a practical way of testing this but I'm open to suggestions (I have tried it on a private channel to make sure it at works in it's current form at least).

Reply:
real testing even better!